File: pbworkspacewcl/src/app/notifiche-via-mail/commons/vo/bando-progetti-vo.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { ProgettoNotificheVO } from "./progetto-notifiche-vo";

export class BandoProgettiVO {
    constructor(
        public nomeBandoLinea: string,
        public progrBandoLineaIntervento: number,
        public progetti: Array<ProgettoNotificheVO>,
        public isExpanded?: boolean,
        public allSelected?:boolean
    ) { }
}

File: pbworkspace/pbworkspace-jar/src/main/java/it/csi/pbandi/pbworkspace/integration/vo/BandoProgettiVO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbworkspace.integration.vo;

import java.util.List;

public class BandoProgettiVO {
	private String nomeBandoLinea;
	private Long progrBandoLineaIntervento;
	private List<ProgettoNotificheVO> progetti;

	public String getNomeBandoLinea() {
		return nomeBandoLinea;
	}

	public void setNomeBandoLinea(String nomeBandoLinea) {
		this.nomeBandoLinea = nomeBandoLinea;
	}

	public Long getProgrBandoLineaIntervento() {
		return progrBandoLineaIntervento;
	}

	public void setProgrBandoLineaIntervento(Long progrBandoLineaIntervento) {
		this.progrBandoLineaIntervento = progrBandoLineaIntervento;
	}

	public List<ProgettoNotificheVO> getProgetti() {
		return progetti;
	}

	public void setProgetti(List<ProgettoNotificheVO> progetti) {
		this.progetti = progetti;
	}

}
